src/allmydata/scripts/tahoe_invite.py

```diff
@@ -0,0 +1,105 @@
+import json
+from os.path import join
+from ConfigParser import NoSectionError
+
+from twisted.python import usage
+from twisted.internet import defer, reactor
+
+from wormhole import wormhole
+
+from allmydata.util import configutil
+from allmydata.util.encodingutil import argv_to_abspath
+
+
+class InviteOptions(usage.Options):
+    synopsis = "[options] <nickname>"
+    description = "Create a client-only Tahoe-LAFS node (no storage server)."
+
+    optParameters = [
```

Comment:
For the sake of being consistent with the standard `tahoe create-client`/`tahoe create-node` options, these should probably be changed throughout to `--shares-needed`, `--shares-happy`, and `--shares-total` (instead of just `--needed`, `--happy`, and  `--total`).